components/usage/pkg/db/workspace_instance_test.go

```diff
@@ -18,66 +18,67 @@ import (
 func TestListWorkspaceInstancesInRange(t *testing.T) {
 	conn := dbtest.ConnectForTests(t)
 
-	workspaceID := "gitpodio-gitpod-gyjr82jkfnd"
+	workspace := dbtest.CreateWorkspaces(t, conn, dbtest.NewWorkspace(t, db.Workspace{}))[0]
+
 	valid := []db.WorkspaceInstance{
 		// In the middle of May
 		dbtest.NewWorkspaceInstance(t, db.WorkspaceInstance{
-			WorkspaceID:  workspaceID,
+			WorkspaceID:  workspace.ID,
```

Comment:
Was this change necessary or just tidying up?

Reply:
To ensure the joining works correctly, we needed to create an actual Workspace record in the DB (line 21). That Workspace gets a random ID each time, so it's references here.

Aside, getting a random ID is a good thing, as it allows tests against the DB to run concurrently without interfering with each other.